var/spack/repos/builtin/packages/mgis/package.py

```diff
@@ -64,6 +65,18 @@ class Mgis(CMakePackage):
     depends_on('boost+python+numpy', when='+python')
     extends('python', when='+python')
 
+    def patch(self):
```

Comment:
Very nice. I did not know that we could embedded tests ! Super cool. Spack rules !